import/cypher/parralel_import_with_mgconsole/README.md

```diff
@@ -0,0 +1,57 @@
+
+# Multiprocess import with Memgraph Example 
+
+This example demonstrates how to split your Cypher file into nodes and relationships and how to use multiple processes to load data efficiently.
+
+
+## 🧠 What This Example Does
+
+The script performs the following actions:
+
+1. **Run cypher_file_splitter_script.py** - helper script to split pokec dataset into nodes and relationships and saving into sperate folder.
+2. **Run multiprocess_import_test.py**
+   - you should create proper indices before hand in this case it is:
+      -`CREATE INDEX ON :User;`
+      -`CREATE INDEX ON :User(id);`
+   - this script first loads nodes than relationships and uses 8 processes for parallel import
+
```

Comment:
also the script can execute `STORAGE MODE IN_MEMORY_ANALYTICAL`, `DROP GRAPH` and creation of indices